### ISSUE TITLE
Group status inactive/paused reason

### ISSUE THREAD

**Issue by mikeallbutt** [DiscipleTools/disciple-tools-theme#810]
So that metrics better reflect the situation, add a way to assign the reason for a group to be inactive in the group status. There are only 2 choices: Inactive, Active. 

Maybe the [Group] is still meeting but not being updated or are away travelling.

This setting [reason for group on pause - inactive] could also disable the update needed flag.

This issue might relate to issue #777 where a new group is about to be formed but hasn't actually started (which would also be a good reason to have the ability to set the start date of a group).

**Comment by corsacca** [comment]
@mikeallbutt I think you are suggestion adding `Paused` as a default status.
And also adding a `reason paused` and a `reason inactive` field.

You you list out a few `reason paused` and a few `reason inactive` ?

**Comment by mikeallbutt** [comment]
The more I use DT the more I need this Group Status. I want to have accurate data, but with only 2 options at present this is more complicated. If I set a group as "Inactive" because they are not meeting for a while but will resume in the future, then DT applies a closed data to the group. When the group is reactivated, the closed date is still set too.

**Comment by mikeallbutt** [comment]
It is possible to do this now via the WP Admin > Settings DT > Custom Fields > Group Status.

**Comment by mikeallbutt** [comment]
Yes it is possible to create different group statuses (like `Paused`) but there is no way to describe why the status has been set. This would be helpful - like what reasons where given previously in this thread.
